aaronpeikert/TDLM.jl
# restructure

- add skeleton and first function
- enable CI on devel
- add rand_cov
- use fixed environment for tests
- run doctests during testing and not documentation
- add a commented version of decoder training
- fix link
- dont copy manifest
- improve script
- force symmetry in `rand_cov`
- type from Distributions is required
- add basic implementation of RandomSequence
- increase coverage
- should test for the type, not the object
- fix non ending doctest
- subtyping got me again...
- add PatterSequence
- doctest for RandomSequence never ran
- rng should be kwarg
- only one parameter really needed
- update doctests
- load simulate in doctests
- tickle CI
- make PatternSequence more flexible
- add RandomLength
- cant be int...
- move random length to utilities
- backup
- Revert "backup"
- dont explicitly load
- restructure
